### DISCUSSION TITLE
fastlane build issues on CircleCI: "There are no local code signing identities found"

### DISCUSSION THREAD

**Discussion by lillybeans** [fastlane/fastlane#17392] [Q&A] [answered]
Hi everyone,

We are trying to build an iOS application on CircleCI, and about a month ago we started running into build issues. Here are the relevant logs after `bundle exec fastlane`

[16:01:11]: 🔓  Successfully decrypted certificates repo
[16:01:11]: Installing certificate...
security: SecKeychainSearchCopyNext: The specified item could not be found in the keychain.
**[16:01:12]: There are no local code signing identities found.**
You can run `security find-identity -v -p codesigning fastlane_tmp_keychain` to get this output.
This Stack Overflow thread has more information: https://stackoverflow.com/q/35390072/774.
(Check in Keychain Access for an expired WWDR certificate: https://stackoverflow.com/a/35409835/774 has more info.)
[16:01:12]: Setting key partition list... (this can take a minute if there are a lot of keys installed)
[16:01:12]: security: SecItemCopyMatching: The specified item could not be found in the keychain.
[16:01:12]: Setting key partition list... (this can take a minute if there are a lot of keys installed)

We use match to pull all the certs and provisioning profiles stored in a private git repo.

We are able to build our apps just fine on local machines, and the problem only exists when we try to build on CircleCI. However, I'd like to understand under what circumstances does fastlane produce these errors, and how should we address them?

**Comment by lillybeans** [comment, marked as answer]
Nevermind - this is not the error causing the build issues. Could someone close this discussion?

**Reply by brad** [reply]
What was the cause?

**Comment by rogerluan** [comment]
@max-ott is there a way to close Discussions? 🙇 Appreciate it! 

EDIT: Nevermind, there isn't, I just marked the comment above as the answer 😊  not gonna lock this discussion.
